app/src/main/java/io/appium/uiautomator2/utils/JSONUtils.java

```diff
@@ -57,4 +64,44 @@ public static Integer readInteger(JSONObject payload, String name, boolean isReq
         }
         return null;
     }
+
+    public static Map<String, Object> toMap(JSONObject obj) {
```

Comment:
Nullable annotation?

Reply:
the result will never be null. It can be an empty map in the worst case. Same below

Reply:
Ah, okay. I thought null since they returned null when obj == null